README.md

```diff
@@ -61,13 +66,13 @@ docker exec -it cassandra cqlsh
 Then, in `cqlsh` you create an `akka_streams` keyspace:
 
 ```cql
-CREATE KEYSPACE akka_streams WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 3 };
+CREATE KEYSPACE akka_streams WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : '1' };
```

Comment:
I guess this is a habit from some earlier versions ... will correct this to integer